### ISSUE TITLE
Baritone for 1.21.1?

### ISSUE THREAD

**Issue by OhNo2** [cabaletta/baritone#4481]
## What do you need help with?
With as much detail as possible, describe your question and what you may need help with.
When will baritone work for 1.21.1? I found the 1.21 version but it doesn't work with fabric 1.21.1. 

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by JPTV-21** [comment]
Yea we need Baritone for 1.21.1

**Comment by ZacSharp** [comment]
See #4466

**Comment by OwenRossing** [comment]
i compiled an updated version, all that needed to change was around one line of code that tells the mod loader what version of minecraft its capable of running on, in this case it was 1.20.4 but someone pointed this out in issue #4466 and the creator quickly changed it but didnt compile and release it (unless im blind), i took the liberty of doing it myself cuz im impatient: https://drive.google.com/drive/folders/1NF4FnlCz3fbPlPygYzONufbAUihyjdSf?usp=sharing

heres my video explaining and showcasing it: https://www.youtube.com/watch?v=H1m3Ovf8pEQ

**Comment by ZacSharp** [comment]
No, there is no release yet (as usual). However there are automated builds for (almost) every commit made in this repository. E.g. [here](https://github.com/cabaletta/baritone/actions/workflows/gradle_build.yml?query=is%3Asuccess+branch%3A1.21+event%3Apush) is the list of all CI builds on the 1.21 branch. If you select one made after merging #4466 (any but the oldest one) and download the artifacts file at the bottom of the page you get a zip containing the usual release files.

You can even change that one line of metadata in the built jar:
1. Rename to ".zip" in case your system can't open the jar directly
2. Open fabric.mod.json (fabric) or META-INF/mods.toml (forge) inside the mod and fix it
3. Rename back to ".jar"

Neoforge doesn't need any changes.

**Comment by LilArxback** [comment]
ı am turks

**Comment by ZacSharp** [comment]
There's been a release quite some time ago (iirc) and now there is [v1.11.3](https://github.com/cabaletta/baritone/releases/tag/v1.11.3) as well, so time to close this.
